programs/bpf/rust/noop/src/solana_sdk.rs

```diff
@@ -0,0 +1,211 @@
+//! @brief Solana Rust-based BPF program utility functions and types
```

Comment:
Looks like this module needs a better home and lots of unit tests

Reply:
Yup, mentioned as part of the 'needs to happen' stuff here: 
https://github.com/solana-labs/solana/issues/2268.  Do you want that to happen before landing this PR?

Reply:
Yes, unit tests should come in alongside new code (or prior!)

Reply:
:-). Added now